CREDITS

```diff
@@ -1,3 +0,0 @@
-;; scoutapm
-Chris Schneider <chris@scoutapm.com> (lead)
-James Titcumb [asgrim] <james@asgrim.com> (lead)
```

Comment:
I suggest to keep the file, but only to remove the email addresses (or even only the angle brackets around those). Also, the format of CREDITS is:
* 1st line: name of the extension (without the leading `;;`)
* 2nd line: comma delimited list of maintainers

So this could be something like:
````
scoutapm
Chris Schneider (lead), James Titcumb [asgrim] (lead)
````

I don't know how the file is used on non Windows architectures, but there it is used to add meta information to the DLL, and it is also contained in the official package builds.

Reply:
Thanks - updated